initial_settings/initial.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python -tt
+
+import subprocess
+from sys import argv
+import dnf
+
+
+class DnfUpgrageTo():
+    def create_repo(self):
```

Comment:
should not be needed, see https://github.com/jsilhan/richdeps-docker/pull/2/files#r45382081